wolfcrypt/test/test.c

```diff
@@ -45669,6 +45669,171 @@ static wc_test_ret_t dilithium_param_test(int param, WC_RNG* rng)
 }
 #endif
 
+/* Tests decoding a key from DER without the security level specified */
+static wc_test_ret_t test_dilithium_decode_level(const byte* rawKey,
```

Comment:
./configure --disable-shared  --enable-dilithium=verify,44,65,87
make

Bunch of errors.

Reply:
fixed